todoapp/src/components/Todo.tsx

```diff
@@ -168,13 +184,83 @@ const useStyles = makeStyles({
         }
       }
     }
-  }
+  },
 })
```

Comment:
if styles are too big like this, you need to create a separate file for this and then import here and here.